Neos.Cache/Classes/Backend/FileBackend.php

```diff
@@ -250,7 +250,11 @@ public function findIdentifiersByTag(string $searchedTag): array
             if ($expiryTime !== 0 && $expiryTime < $now) {
                 continue;
             }
-            if (!in_array($searchedTag, explode(' ', (string)substr($metaData, 0, -self::EXPIRYTIME_LENGTH)))) {
+           $extractedTags = substr($metaData, 0, -self::EXPIRYTIME_LENGTH);
+           if ($extractedTags === false) {
```

Comment:
```suggestion
           if ($extractedTags === false || !in_array($searchedTag, explode(' ', $extractedTags))) {
```
would be my way of doing it. But that's just style…

Reply:
Good point! Lets do that. I _think_ though if I commit your suggestion it  contains only that line and not the  extractedTags